server/index.js

```diff
@@ -46,6 +46,7 @@ shutdown(
 app.use(
   (req, _, next) => {
     req.context = require('../data.json');
+    req.states = require('../states.json');
```

Comment:
I'd like to keep `context` as the only thing we pass around to page components. Perhaps the pulling in the state information could be:

```suggestion
req.context = {
  candidates: require('.../data.json'),
  states: require('.../states.json')
};
```

and within `home` we update its use of context?